policy_validator.c

```diff
@@ -14,6 +16,15 @@ static inline uint64_t policy_validator_reg_reader(uint32_t reg_num)
     return (uint64_t)policy_validator_hack_env->gpr[reg_num];
 }
 
+static inline uint64_t policy_validator_address_fixer(uint64_t vaddr)
```

Comment:
Shouldn't this be target_ulong since a RV32 can have virtual memory too?

Reply:
Most of the functions in this file treat the addresses as 64-bit, so I think this is more consistent.

Reply:
Right.  That way the interface is consistent.